pulsemeeter/interface/main_window.py

```diff
@@ -250,22 +253,31 @@ def start_vumeters(self):
                 self.vu_list[device_type][device_id] = vumeter
 
     def start_app_list(self):
+        # this is probably not the best solution but it handles the pactl errors fine
         sink_input_viewport = self.builder.get_object('sink_input_viewport')
         source_output_viewport = self.builder.get_object('source_output_viewport')
         try:
             self.sink_input_box = AppList('sink-input', self.client)
             self.source_output_box = AppList('source-output', self.client)
+
+            sink_input_viewport.add(self.sink_input_box)
+            source_output_viewport.add(self.source_output_box)
+
+            self.subscribe_thread = threading.Thread(target=self.listen_subscribe, args=())
+            self.subscribe_thread.start()
         except Exception as ex:
-            print('App sinks returned an error, audio backend probably crashed')
+            print('App sinks returned an error, audio backend probably crashed. Server will be closed.')
             if self.windowinstance is not None:
                 self.windowinstance.destroy()
             self.delete_event()
+            try:
```

Comment:
the ui should not stop the server if there's an issue with pulse, just let the client close itself. If pulse crashes than the server has to catch that exception and handle it instead of just closing